frontend/src/bundles/common/services/toast/toast.service.ts

```diff
@@ -0,0 +1,92 @@
+import { type createStandaloneToast } from '@chakra-ui/react';
+
+type Constructor = {
+    toast: ReturnType<typeof createStandaloneToast>['toast'];
+};
+
+class ToastService {
+    private toast: ReturnType<typeof createStandaloneToast>['toast'];
+
+    public constructor({ toast }: Constructor) {
+        this.toast = toast;
+    }
+
+    public warn = (message: string, toastId: string, title: string): void => {
+        this.toast({
+            id: toastId,
+            title: title,
+            description: message,
+            status: 'warning',
+            duration: 7000,
+            isClosable: true,
+            position: 'top-right',
+            variant: 'solid',
+        });
```

Comment:
You are using the same properties for every method except for status.

Create a separate function for calling toast and just pass status as argument

Reply:
done